nilrt-logging.bb: Add recipe to install INI file to specify log paths

pratheekshasn opened the pull request:
### Summary of Changes

An INI file is added at /usr/share/ni-resetniconfig/logging_paths.ini that contains a list of file paths and directory paths. The contents of these paths then get added to the technical report that can be generated through Hardware Config Utility.

### Justification

[AB#3110391](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/3110391)


### Testing

I have verified that the INI file is installed at the path /usr/share/ni-resetniconfig/logging_paths.ini upon installing base-system-image. The contents of the file are also present as specified in the file next to the recipe.

* [x] I have built the core package feed with this PR in place. (`bitbake packagefeed-ni-core`)


### Procedure

* [x] I certify that the contents of this pull request complies with the [Developer Certificate of Origin](https://github.com/ni/nilrt/blob/HEAD/docs/CONTRIBUTING.md#developer-certificate-of-origin-dco).

chaitu236 commented on recipes-core/packagegroups/packagegroup-ni-runmode.bb:
```diff
@@ -24,6 +24,7 @@ RDEPENDS:${PN} = "\
 	libyaml \
 	linux-firmware-radeon \
 	lldpd \
+	lvrt-logging \
```
This logging is not specific to lvrt. Rename to `nilrt-logging` or something else.

pratheekshasn replied:
I have fixed this now.

chaitu236 commented on recipes-support/lvrt-logging/files/logging_paths.ini:
```diff
@@ -0,0 +1,7 @@
+[Dirs]
+Dir1 = /var/log
+Dir2 = /var/local/natinst/log
+Dir3 = /var/lib/pstore
```
Missing newline at end

![image](https://github.com/user-attachments/assets/8333bed4-a189-4f08-883c-e4dd4ccd6f53)

pratheekshasn replied:
I have fixed this now.

chaitu236 commented on recipes-support/lvrt-logging/lvrt-logging.bb:
```diff
@@ -0,0 +1,11 @@
+SUMMARY = "INI file to specify logging paths for LVRT"
```
INI file is a implementation detail that doesn't make sense to go in package summary. Just say `Configuration for NILRT Logging` or such.

pratheekshasn replied:
I have fixed this now.

chaitu236 reviewed (changes requested):
Follow 50/72 rule in commit message

chaitu236 commented on recipes-support/lvrt-logging/files/logging_paths.ini:
```diff
@@ -0,0 +1,7 @@
+[Paths]
+File1 = /README_File_Paths.txt
```
Remove

pratheekshasn replied:
Is it still okay to keep the section?

chaitu236 replied:
No, better remove

pratheekshasn replied:
Done.

pratheekshasn commented:
I have rewritten the commit message now.

pratheekshasn committed:
nilrt-logging.bb: Config file to list log paths

A config file is added at /usr/share/ni-resetniconfig/logging_paths.ini
that contains a list of file paths and directory paths. The
contents of these paths then get added to the technical report
that can be generated through Hardware Config Utility.

Signed-off-by: pratheekshasn <pratheeksha.s.n@ni.com>
